nextflow_schema.json

```diff
@@ -694,6 +694,214 @@
                     "fa_icon": "fas fa-head-side-mask",
                     "description": "Specify the minimum dust score for PRINTSEQ++ complexity filtering",
                     "help_text": "Specify the minimum dust score below which low-complexity reads will be removed. A DUST score is based on how often different tri-nucleotides occur along a read.\n\n> Modifies tool parameter(s):\n> - PRINSEQ++: `--lc_dust`"
+                },
+                "metagenomics_profiling_tool": {
+                    "type": "string",
+                    "description": "Specify which tool to use for metagenomic profiling and screening. Required if `--run_metagenomics` flagged.",
+                    "enum": ["malt", "metaphlan", "kraken2", "krakenuniq"],
+                    "fa_icon": "fas fa-toolbox",
+                    "help_text": "Select which tool to run metagenomics profiling on designated metagenomics_screening_input. These tools behave vastly differently due to performing read profiling using different methods and yield vastly different reuslts."
```

Comment:
Could give general guidance? e.g. MALT good for getting aDNA authentication but requires lots of mem